scripts/provisioning/s3_haproxy_config.py

```diff
@@ -221,4 +221,3 @@ def process(self):
     os.system("rm -rf /etc/cron.daily/logrotate")
     os.system("cp /opt/seagate/cortx/s3/install/haproxy/logrotate/logrotate /etc/cron.hourly/logrotate")
     os.system("systemctl restart rsyslog")
-    os.system("systemctl restart haproxy")
```

Comment:
1. Need to check an alternative for audit log - rsyslog configuration.
2. Need to communicate HA about CORTXHA haproxy reload.
3. Need to handle haproxy rsyslog configuration.

Reply:
1. No audit log configuration in config phase.
2. Communicate about reload interface with HA team. (Have alternative in the wrapper to switch to systemctl if reload interface not there, so no failures)
3. Communicate about rsyslog configuration with provisioner team. (rsyslog changes not removed, need to wait for provisioner to incorporate those changes)
Can be merged : Working and tested.